### PR TITLE
19190 Fixed "return to MBR" issues

### PR DESCRIPTION
**Pull request by severinbeauvais**
*Issue #:* bcgov/entity#19190

*Description of changes:*
- app version = 7.0.33
- only show "return to MBR" for temp amalgamations
- only show "return to MBR" for temp incorporation applications
- only show "return to MBR" for temp registrations

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the business-filings-ui license (Apache 2.0).

### CODE REVIEW

File: src/components/Dashboard/FilingHistoryList/filings/AmalgamationFiling.vue [hunk 1]
```diff
@@ -32,7 +32,7 @@
       />
 
       <div
-        v-else-if="isStatusCompleted"
+        v-if="!!tempRegNumber && isStatusCompleted"
```
Comment (severinbeauvais) [inline src/components/Dashboard/FilingHistoryList/filings/AmalgamationFiling.vue:35]: Sample:

image

File: src/components/Dashboard/FilingHistoryList/filings/IncorporationApplication.vue [hunk 2]
```diff
@@ -32,7 +32,7 @@
       />
 
       <div
-        v-else-if="isStatusCompleted"
+        v-if="!!tempRegNumber && isStatusCompleted"
```
Comment (severinbeauvais) [inline src/components/Dashboard/FilingHistoryList/filings/IncorporationApplication.vue:35]: Sample:

image

File: src/components/Dashboard/FilingHistoryList/filings/RegistrationFiling.vue [hunk 3]
```diff
@@ -6,14 +6,18 @@
   >
     <template #body>
       <div
-        v-if="isStatusCompleted"
+        v-if="!!tempRegNumber && isStatusCompleted"
```
Comment (severinbeauvais) [inline src/components/Dashboard/FilingHistoryList/filings/RegistrationFiling.vue:9]: Sample:

image

**Review by JazzarKarim — APPROVED** [review]

File: src/components/Dashboard/FilingHistoryList/filings/IncorporationApplication.vue (hunk 2 above)
Comment (severinbeauvais) [reply src/components/Dashboard/FilingHistoryList/filings/IncorporationApplication.vue:35]: Sample using Txxx id:

image

File: src/components/Dashboard/FilingHistoryList/filings/RegistrationFiling.vue (hunk 3 above)
Comment (severinbeauvais) [reply src/components/Dashboard/FilingHistoryList/filings/RegistrationFiling.vue:9]: Sample using Txxx id:

image

**Comment by severinbeauvais** [comment]
I asked Yui to review these screenshots and he says they look good.

**Pull request merged**